helm/Chart.yaml

```diff
@@ -14,3 +14,4 @@ dependencies:
   - name: kafka-topic-creator
     repository: "https://storage.googleapis.com/hypertrace-helm-charts"
     version: 0.1.9
+    condition: not hypertraceAlertEngine.enabled
```

Comment:
We don't need a topic if not using the aggregated form? The individual deployments should still be using kafka I thought

Reply:
The aggregate service doesn't use kafka

Reply:
Updated